Src/ST-LIB_HIGH/Protections/Boundary.cpp

```diff
@@ -0,0 +1,16 @@
+#include "ST-LIB_HIGH/Protections/Boundary.hpp"
+
+ const map<type_id_t,uint8_t> BoundaryInterface::format_look_up{
+    {type_id<int>,0},
```

Comment:
Maybe using an Enum here for clariry

Reply:
the map gives me direct access, instead of having to check the type in a switch, I cannot use an enum with the same functionality, also I'd say it is quite readable